{"bookmarked": 3, "bucket_name": "Today", "bucket_order": 3, "change_log": [{"anon": "stud", "data": "jfd2i9d3ni47fe", "type": "create", "uid_a": "a_0", "when": "2018-03-29T22:05:24Z"}, {"anon": "no", "data": "jfd2y04o8j52d4", "to": "jfd2i9d0a7y7fd", "type": "i_answer", "uid": "icdxydvnk125h8", "when": "2018-03-29T22:17:38Z"}, {"anon": "stud", "to": "jfd2i9d0a7y7fd", "type": "followup", "uid_a": "a_0", "when": "2018-03-30T05:47:54Z"}], "children": [{"bucket_name": "Today", "bucket_order": 3, "children": [], "config": {}, "created": "2018-03-29T22:17:38Z", "data": {"embed_links": []}, "folders": [], "history": [{"anon": "no", "content": "<p>hmm. is this happening to anyone else?3</p>", "created": "2018-03-29T22:17:38Z", "subject": "", "uid": "icdxydvnk125h8"}], "history_size": 1, "id": "jfd2y04d7a2d3", "is_tag_endorse": false, "tag_endorse": [], "tag_endorse_arr": [], "type": "i_answer"}, {"anon": "stud", "bucket_name": "Week 3/25 - 3/31", "bucket_order": 157, "children": [], "config": {}, "created": "2018-03-30T05:47:54Z", "data": {"embed_links": null}, "folders": [], "id": "jfdj11w7hjf1iy", "no_answer": 0, "no_upvotes": 0, "subject": "<p>Seems to have been some transient issue--it&#39;s working fine for me now.</p>", "tag_good": [], "tag_good_arr": [], "type": "followup", "uid_a": "a_0", "updated": "2018-03-30T05:47:54Z"}], "config": {}, "created": "2018-03-29T22:05:24Z", "data": {"embed_links": []}, "default_anonymity": "no", "drafts": null, "folders": ["faculty_advising", "eecs"], "history": [{"anon": "stud", "content": "<p>When I try to access the faculty advising web app, I get an error message saying that a database query failed:</p>\n<p><img src=\"https://d1b10bmlvqabco.cloudfront.net/attach/hyq0br1u3kx7dg/icguxy0896q1j8/jfd2c6syw2y1/advising_bug.PNG\" alt=\"\" /></p>\n<p>Should I be concerned about this, or is it a general problem everyone is having?<br /><br />(I&#39;ve tried logging out and back in again and opening it in an incognito window, but neither helped)</p>", "created": "2018-03-29T22:05:24Z", "subject": "Faculty advising app bug", "uid_a": "a_0"}], "history_size": 1, "i_edits": [], "id": "jfd2i9d0a7y7fd", "is_bookmarked": false, "is_tag_good": false, "my_favorite": false, "no_answer": 0, "no_answer_followup": 0, "nr": 6417, "num_favorites": 0, "q_edits": [], "request_instructor": 0, "request_instructor_me": false, "s_edits": [], "status": "active", "t": 1654549901688, "tag_good": [], "tag_good_arr": [], "tags": ["eecs", "faculty_advising", "student"], "type": "question", "unique_views": 279}